<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFigureDialog</class>
 <widget class="QDialog" name="NewFigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>141</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="buttonKnight">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/knightWhite</normaloff>:/img/knightWhite</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>110</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonBishop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/bishopWhite</normaloff>:/img/bishopWhite</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>110</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRook">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/rookWhite</normaloff>:/img/rookWhite</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>110</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonQueen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/queenWhite</normaloff>:/img/queenWhite</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>110</width>
       <height>110</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
